content/updates/2020-06/2-option-screen-revamp.md

```diff
@@ -7,12 +7,12 @@ pim_announcement_audience:
 ---
 
 # Better manage your attribute options
-::: meta-data type="Improvement" features="Attribute Options, Productivity, Data Governance" available="late June" link-to-doc="../articles/manage-your-attributes.html#manage-simple-and-multi-selects-attribute-options"
+::: meta-data type="Improvement" features="Data Governance" available="late June" link-to-doc="../articles/manage-your-attributes.html#manage-simple-and-multi-selects-attribute-options"
```

Comment:
I would keep "Productivity" for this one.

Reply:
fixed :) thanks!